sdk/protocol-plugins/src/plugins/aave-v3/interfaces/IAaveV3LendingPool.ts

```diff
@@ -1,8 +1,51 @@
-import { ILendingPool } from '@summerfi/sdk-common/protocols'
-import { IAaveV3DebtConfigMap } from './IAaveV3DebtConfigMap'
-import { IAaveV3CollateralConfigMap } from './IAaveV3CollateralConfigMap'
+import { ILendingPool, ILendingPoolData, LendingPoolSchema } from '@summerfi/sdk-common/protocols'
+import { z } from 'zod'
+import {
+  AaveV3LendingPoolIdSchema,
+  IAaveV3LendingPoolId,
+  IAaveV3LendingPoolIdData,
+} from './IAaveV3LendingPoolId'
 
-export interface IAaveV3LendingPool extends ILendingPool {
-  collaterals: IAaveV3CollateralConfigMap
-  debts: IAaveV3DebtConfigMap
+/**
+ * @interface IAaveV3LendingPoolData
+ * @description Represents a lending pool in the Aave V3 protocol
+ */
+export interface IAaveV3LendingPoolData extends ILendingPoolData {
+  /** The lending pool's ID */
+  readonly id: IAaveV3LendingPoolIdData
 }
+
+/**
+ * @interface IAaveV3LendingPool
+ * @description Interface for the implementors of the lending pool
+ *
+ * This interface is used to add all the methods that the interface supports
+ */
+export interface IAaveV3LendingPool extends ILendingPool, IAaveV3LendingPoolData {
+  readonly id: IAaveV3LendingPoolId
+}
+
+/**
+ * @description Zod schema for IAaveV3LendingPool
+ */
+export const AaveV3LendingPoolSchema = z.object({
+  ...LendingPoolSchema.shape,
+  id: AaveV3LendingPoolIdSchema,
+})
```

Comment:
Will you refactor it later to infer types from schemas instead of declaring it twice?

Reply:
Yes, indeed. I've done for all common types no in the Morpho branch but I didn't want to touch Aave v3 or Spark as you are working on those. Let's catch up on Monday about it